### ISSUE TITLE
deux plantages d'affilée...

### ISSUE THREAD

**Issue by kaori01** [JulienNigon/SpellcasterUniversity-Bug-Report#814]
Première fois que cela m'arrive, deux fois de suite le jeu freeze... Je suis sur la carte du ménestrel.

![image](https://user-images.githubusercontent.com/60629516/74386541-82349880-4df6-11ea-8187-bc6f51630f2c.png)

**Comment by kaori01** [comment]
et de trois, cette fois juste après avoir obtenu une carte totem via le deck nature...

**Comment by kaori01** [comment]
cette partie doit vraiment avoir un problème, je vais chercher si j'arrive à trouver le fichier de sauvegarde (je ne suis pas douée lol!)

En attendant, plantage cette fois parce que j'ai consommé une mandragore pour commencer une nouvelle potion...

![image](https://user-images.githubusercontent.com/60629516/74424418-34e61480-4e52-11ea-89d1-af960c092e4c.png)

**Comment by kaori01** [comment]
trouvé (heureusement que je donne des noms débiles à mes universités!) mais le format n'est pas supporté ici... 😭 Enfin, voilà, j'ai copié le fichier XML sur mon bureau, si vous le voulez comme ça je pourrai vous le ressortir...

**Comment by Florane88** [comment]
Ça m'est arrivé aussi ce matin (vers 7h), très peu de temps après avoir lancé une partie enregistrée.
Je ne sais plus ce que je faisais à ce moment-là...

**Comment by Neelan22** [comment]
Oui, je soupçonne un problème avec la sauvegarde. Si tu peux nous l'envoyer, ça nous aiderait beaucoup, merci !

**Comment by kaori01** [comment]
Le format n'est pas accepté ici, je vous l'envoie où? ;)

**Comment by kaori01** [comment]
j'ai trouvé un email en farfouillant... julien.nigon@orange.fr J'y ai envoyé le fichier...

**Comment by Neelan22** [comment]
oui c'est bon merci, ça nous sera très utile. Désolée, j'ai pris du retard dans les mails...
